README.md

```diff
@@ -20,14 +24,20 @@ Just raise [an issue](https://github.com/COSIMA/cosima-recipes/issues) explainin
 
 ## Contents
 
-### [Tutorials](https://cosima-recipes.readthedocs.io/en/latest/tutorials.html)
+We are in the process of transition these recipes from using [cosima-cookbook](https://github.com/COSIMA/cosima-cookbook) infrastructure to an _intake catalogue_, so you will find recipes may use either method to access model data. If you are halfway through a project using the cosima-cookbook and looking for resources, this [tag](https://github.com/COSIMA/cosima-recipes/tree/cosima_cookbook) marks the repository before most recipes were transitioned.
```

Comment:
let's just remove any references to cookbook... whoever uses cookbook they know, else they should just go to Intake

Reply:
Yeah, I agree...

Reply:
I think it's better to describe where we are at ... there's lots of unresolved references to the cookbook in docs and notebooks that will take some time to work through

Reply:
I prefer we have a clean tutorial for the Intake and how people should open datasets and don't mention cookbook at all. It creates confusion.

Reply:
I opened an issue to discuss this: #425